geth/jail/extensions/console/console.go

```diff
@@ -0,0 +1,141 @@
+package console
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/extensions"
+	"github.com/status-im/status-go/geth/node"
+)
+
+var (
+	// Stdout defines the default writer for the console.log
+	// delivery call.
+	Stdout io.Writer = os.Stdout
+
+	// EventConsoleLog defines the event type for the console.log call.
+	EventConsoleLog = "vm.console.log"
+
+	// EventConsoleWarn defines the event type for the console.debug call.
+	EventConsoleWarn = "vm.console.warn"
+
+	// EventConsoleDebug defines the event type for the console.debug call.
+	EventConsoleDebug = "vm.console.debug"
+
+	// EventConsoleError defines the event type for the console.error call.
+	EventConsoleError = "vm.console.error"
+
+	_ = extensions.Register(func(vm *otto.Otto) error {
+		return vm.Set("console", map[string]interface{}{
+			"log":   consoleLog,
+			"error": consoleError,
+			"debug": consoleDebug,
+			"warn":  consoleWarn,
```

Comment:
Can we simplify this so that only `log` is exposed (the `error`, `debug`, `warn` essentially doing the same thing, thus are redundant).